types/react-native/index.d.ts

```diff
@@ -5827,6 +5921,27 @@ interface PlatformWebStatic extends PlatformStatic {
     OS: 'web';
 }
 
+/**
+ * Select native platform color
+ * The color must match the string that exists on the native platform
+ *
+ * @see https://reactnative.dev/docs/platformcolor#example
+ */
+export function PlatformColor(color: string): string;
+
+type DynamicColorIOSTuple = {
+    light: string;
```

Comment:
This should be ColorValue (at least, that's what it is in Flow, not sure what it is in these TypeScript definitions), as you can use any color, including a PlatformColor:

https://github.com/facebook/react-native/blob/ffa3d7f638c820dc208320193e6ba65667d751eb/Libraries/StyleSheet/PlatformColorValueTypes.ios.js#L28-L31